tests/server/test_environment_metrics.py

```diff
@@ -1406,7 +1406,9 @@ async def test_metric_aggregation_no_date(
     assert result.result["data"]["metrics"]["gauge_metric1"] == [None for _ in range(10)]
 
 
-async def test_cleanup_environment_metrics(init_dataclasses_and_load_schema) -> None:
+@pytest.mark.parametrize("env1_halted", [True, False])
+@pytest.mark.parametrize("env2_halted", [True, False])
+async def test_cleanup_environment_metrics(server, client, env1_halted, env2_halted) -> None:
```

Comment:
This test case didn't start a server before, but now it does by adding the `server` fixture. This means that next to the metrics created by this test case, the Inmanta server will create metrics as well. If that happens the test case will fail. This test case was not built to run while an Inmanta server is running and I think we should keep it like that.